app/views/sales/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Vendor Sales</h1>
+<ul>
+
+    <span><%=@sales.name%></span>
+    <span><%=@sales.vendor_id%></span>
```

Comment:
If `@sales` is a list of many things then you need to `.each` through the list to display them.